### PR TITLE
Fixes #28225 - Update Rubocop to 0.75

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

File: .rubocop_todo.yml
Comment (tbrisker) [inline .rubocop_todo.yml]: These look like cops we fixed but new version found some additional cases that weren't detected before?

Comment (mmoll) [reply .rubocop_todo.yml]: Probably, yes... I'll autocorrect...

**Comment by tbrisker** [comment]
@mmoll does this mean all plugins also need to be updated to match?

**Comment by mmoll** [comment]
@tbrisker I'd say yes, like on every other update. :package:

**Comment by tbrisker** [comment]
@mmoll can you post on discourse about this so plugin maintainers know that they need to prepare for updating rubocop before we merge and break everyone?

**Comment by mmoll** [comment]
https://community.theforeman.org/t/plugin-authors-rubocop-update-incoming/16240

**Review by lzap — APPROVED** [review]

**Comment by lzap** [comment]
https://github.com/theforeman/foreman_remote_execution/pull/447

**Comment by tbrisker** [comment]
Thanks @mmoll !

**Pull request merged**
